qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/common/data_handling/data_model.py

```diff
@@ -196,7 +200,8 @@ def estimate_background(self, ws_name, niter, xwindow, doSGfilter):
             logger.error("Error on arguments supplied to EnggEstimateFocusedBackground: " + str(e))
             ws_bg = SetUncertainties(InputWorkspace=ws_name)  # copy data and zero errors
```

Comment:
I think this also needs an output workspace name specified ` OutputWorkspace=ws_name + "_bg"`, otherwise when ` EnggEstimateFocussedBackground` fails I get an additional workspace `ws_bg` produced.